lib/src/connection.rs

```diff
@@ -242,6 +291,13 @@ impl ConnectionInfo {
                 ));
                 Encryption::Tls
             }
+            "neo4j+ssc" => {
```

Comment:
Thanks for adding the warning here, though you also need to add a match arm for `bolt+ssc`.